aws-site-1-app-vm/terragrunt.hcl

```diff
@@ -0,0 +1,26 @@
+include "root" {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "github.com/jeffgiroux/terraform-f5xc-aws-app-vm.git?ref=v0.0.1"
```

Comment:
will a future version of this module configure HTTP load balancers, etc?

Reply:
I suggest HTTP LB be separate module like "terraform-f5xc-http-lb-frontend" or similar...with dependency on this VM workload. Otherwise this new VM workload module would require a dependency on XC site.

Reply:
This triggers for me the "how many is too many" concern you raised elsewhere.

Reply:
For now, to get things working I will combine app and http lb for initial use case...single module. We can discuss later use cases where an HTTP LB is potentially not needed at deployment time, or might want to be manually created via lab guide walk through. A flag "create http lb yes/no" would work as a possible option.

Reply:
This is fine with me.